rel8.cabal

```diff
@@ -60,6 +55,10 @@ library
     -Wno-monomorphism-restriction
     -Wno-missing-local-signatures
     -Wno-missing-kind-signatures
+    -Wno-missing-role-annotations
+    -Wno-missing-deriving-strategies
+    -Wno-term-variable-capture
```

Comment:
Could we just avoid these warnings rather than disabling them?

Reply:
Seems reasonable. We should probably move away from using `-Weverything` in another PR, since that enables way too many things.